_data/_2020_members.yml

```diff
@@ -28,3 +28,8 @@
   github: samwisebuze
   short_bio: I'm a fourth year CS Major. I currently work as a full-stack web developer for Learn21. I am the president of the UC Game Development Group and server as Webmaster for ACM-W.
   website: https://samuelbuzas.com
+- name: Will Schweitzer
+  image_url: https://avatars3.githubusercontent.com/u/21173745
+  github: williamschweitzer
+  short_bio: I'm a Computer Science major that enjoys making things with code. I am interested in learning to write clean, efficient code in a variety of languages. Primarily C# and Javascript.

```

Comment:
I don't think "Computer Science" is a proper noun, so perhaps it should not be capitalized.

Reply:
Nice catch. I'll change that now.